docs/markdown/simple-switch-based-router.md

```diff
@@ -13,44 +13,45 @@ The example code bellow shows some common usecases
 <!--Reason-->
 
 ```reason
-let handler = request => {
+let handler = Piaf.Server.{request, _} => {
```

Comment:
`Morph`?

Reply:
Yes, this should be a `Morph.Request.t`, I'm not sure what to do with the examples/mini-guides tbh, but I'll update this.